frontend/web/components/import-export/FeatureExport.tsx

```diff
@@ -0,0 +1,215 @@
+  useGetFeatureExportsQuery,
+} from 'common/services/useFeatureExport'
+import InfoMessage from 'components/InfoMessage'
+import FeatureExportItem from './FeatureExportItem'
+
+type FeatureExportType = {
+  projectId: string
+}
+
+const FeatureExport: FC<FeatureExportType> = ({ projectId }) => {
+  const [environment, setEnvironment] = useState<string>(
+    ProjectStore.getEnvs()?.[0]?.api_key,
+  )
+  const [tags, setTags] = useState<(number | string)[]>([])
+  const [search, setSearch] = useState()
+  const [page, setPage] = useState(0)
+  const [tagStrategy, setTagStrategy] = useState<TagStrategy>('UNION')
+
+  useEffect(() => {
+    if (environment) {
+      AppActions.getFeatures(projectId, environment, true, null, null, page, {
+        search,
+        tag_strategy: tagStrategy,
+        tags: tags?.length ? tags : undefined,
+      })
+    }
+  }, [environment, tagStrategy, tags, search, projectId, page])
+  const [createFeatureExport, { isLoading: isCreating, isSuccess }] =
+    useCreateFeatureExportMutation({})
+
+  const { data: exports, refetch } = useGetFeatureExportsQuery({ projectId })
+  const onSubmit = () => {
+    const env = ProjectStore.getEnvironment(environment)
+    createFeatureExport({
+      environment_id: env.id,
+      tag_ids: tags,
+    }).then((res) => {
+      if (res?.data) {
+        toast('Your export is processing')
+      } else {
+        toast('Failed to export features')
+      }
+    })
+  }
+  const timerRef = useRef<NodeJS.Timer>()
+  useEffect(() => {
+    if (
+      exports?.results.find(
+        (featureExport) => featureExport.status === 'PROCESSING',
+      )
+    ) {
+      timerRef.current = setTimeout(() => {
+        refetch()
+      }, 200)
+    }
+  }, [exports, refetch])
+  return (
+    <div className='mt-4'>
+      <InfoMessage>
+        This will export the project's features using the default values from a
+        given environment.
+      </InfoMessage>
+      <div className='mb-2'>
+        <Tooltip
+          title={
+            <>
+              <strong>Environment</strong>
+              <IonIcon icon={informationCircle} />
+            </>
+          }
+        >
+          {
+            'Selecting an environment will determine the feature defaults used when importing to a project'
+          }
+        </Tooltip>
+      </div>
+      <div className='row'>
+        <div className='my-2 col-6 col-lg-4'>
+          <EnvironmentSelect
+            projectId={projectId}
+            onChange={setEnvironment}
+            value={environment}
+          />
+        </div>
+      </div>
+      <div className='my-2'>
+        <Tooltip
+          title={
+            <>
+              <strong>Filter Exported Features</strong>{' '}
+              <IonIcon icon={informationCircle} />
+            </>
+          }
+        >
+          {
+            'If any tags are selected then the exported file will only include features that are tagged.'
+          }
```

Comment:
Maybe some clarity that any feature has any of the supplied tags will be included in the export.